locales/ru.po

```diff
@@ -0,0 +1,774 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"POT-Creation-Date: 2022-02-06 00:12+0100\n"
+"PO-Revision-Date: 2024-06-17 13:29+0300\n"
+"Last-Translator: ssdaniel24 <bo7oaonteg2m__at__mailDOTru>\n"
+"Language-Team: \n"
+"Language: ru\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: pygettext.py 1.5\n"
+"X-Generator: Poedit 3.4.2\n"
+
+msgid "Error: "
+msgstr "Ошибка: "
+
+msgid "Error: I tried to send you an empty message."
+msgstr "Ошибка: я пытался отправить вам пустое сообщение."
+
+msgid "Missing \"%s\".  You may want to quote your arguments with double quotes in order to prevent extra brackets from being evaluated as nested commands."
+msgstr "Отсутствует \"%s\". Возможно вы захотите заключить аргументы в двойные кавычки, чтобы предотвратить считывание дополнительных скобок как вложенных команд."
+
+msgid "\"|\" with nothing preceding.  I obviously can't do a pipe with nothing before the |."
+msgstr "\"I\", перед которым ничего не стоит. Я не могу создать конвейер, если перед этим символом ничего не написано."
+
+msgid "Spurious \"%s\".  You may want to quote your arguments with double quotes in order to prevent extra brackets from being evaluated as nested commands."
+msgstr "Ложный \"%s\". Возможно вы захотите заключить аргументы в двойные кавычки, чтобы предотвратить считывание дополнительных скобок как вложенных команд."
+
+msgid "\"|\" with nothing following.  I obviously can't do a pipe with nothing after the |."
+msgstr "\"I\", за которым ничего не стоит. Я не могу создать конвейер, если после этого символом ничего не написано."
+
+msgid "%s is not a valid %s."
+msgstr "%s не является допустимым %s."
+
+msgid "That's not a valid %s."
+msgstr "Это не является допустимым %s."
+
+msgid "(XX more messages)"
+msgstr "(ещё XX сообщений)"
+
+msgid "more message"
+msgstr "сообщение осталось"
+
+msgid "more messages"
+msgstr "сообщений осталось"
```

Comment:
these two translations need to have fewer bytes bytes as the translation of "XX more messages" minus the leading "XX " (preferably as many), because they are used in the same context. (the number is prepended to these two strings)

Reply:
You don't have to, you just need to replace `(ещё XX сообщений)` with `(XX сообщений осталось)`, so that both translations have the same number of bytes in the end